### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by bojunl** [mchao409/ORIE-4741-Class-Project#16]
This project aims to predict the number of shared-bikes that will be rented from a location in San Francisco within a certain period. This helps bike renters choose the time to rent bikes as well as bike-sharing companies to distribute bikes to different locations. The dataset comes from San Francisco Bay Area Bike Share data and includes information like rental location, weathers, etc.

Things I like about this project:

1.	I think the team did a great job cleaning data. For example, when identifying corrupted zip codes, instead of dropping the data, they chose to use the Google API to find the correct entries.
2.	Another thing that impressed me is that they chose to analyze the problem with different models (like regularized linear models, decision trees, etc.) and also tried out different loss functions and hyperparameters.
3.	Finally, the description of how data cleaning, feature engineering processes work is very clear and easy to follow. The report also gave details reasons for why they chose to conduct the data manipulations in such ways.

Potential areas for improvement:

1.	It would be better if your report contains more graphs to visualize the results, as this would help us understand how your models perform.
2.	Also, I noticed that the R squared of your last linear regression model is only 0.33. I think it’d better if you can explain further what some potential reasons could be - like did you test whether there was any problem with the dataset itself or were there any outliers, etc.
3.	I think the reason you explained why you chose 15 as the maximum decision tree is not sufficient. Moreover, 15 maximum depth does not guarantee an optimal generalization performance. 
4.	Accuracy is not the only way to tell whether the model is good or not. In particular, for this task, measures like precision and recall rates may also worth the discussion.
